### ISSUE TITLE
only the gyroscope should rotate the shell UI

### ISSUE THREAD

**Issue by pixelr0** [ubports/ubuntu-touch#322]
as it is now, launching an app that requires landscape  (ex: Monster Wars) will rotate the shell UI even if you are holding the phone in portrait mode. this is highly confusing when trying to get into the app switcher since the UI is rotated and swiping from right to left does nothing and if the app is in full screen is even worse because. this also ignores the Rotation Lock setting.

**Comment by ChloeWolfieGirl** [comment]
I think he's supposing that the app should only work landscape but the rest should rotate with the phone.

The issue here is that, that's a lot of code to try and figure out, change and solve.

**Comment by UniversalSuperBox** [comment]
The same situation occurs on Android or iOS when a landscape-only app is launched. The only change on these operating systems is the buttons always try to stay in the same place (towards the bottom of the screen on Android, there's only one button on iOS).

Is there another issue other than the UI triggers being in a different place, such as not rotating back once you reach the home screen? If not, I'd call this a design decision.

**Comment by BlueKenny** [comment]
If a app only works in landscape it could control the gyroscope and say "you are in landscape", I think this schould by possible and solve the problem?

**Comment by mateosalta** [comment]
I agree, only the app it's self should rotate - not the shell, which breaks usage.

Test: 
- mount phone in landscape in car
- open navigaion app, this is good and works in landscape
- accept call while driving
- phone changes to dialer app, which is broken in landscape and requests portrait only
- the whole shell rotates to app orientation
- try to exit by swiping from right... instead the recent calls comes up
- now you need to rotate your phone to exit, or even get back to your navigation, a task that should be a quick right swipe to toggle apps 

We need a way of locking only the app orientation and allowing the shell to rotate as expected. aka, top bar always on top no matter what, or locked in the orientation you selected.
